### DISCUSSION TITLE
Add in Combo Bursts

### DISCUSSION THREAD

**Discussion by MineTurtlee** [ppy/osu#30461] [Ideas] [answered]
that might be good, that might be bad according to people's preferences, but it'd be good if it's added in so it looks more like osu! stable (which is still, not enabled by default)

**Comment by cl8n** [comment, marked as answer]
there's a few other discussions about this already, make sure to search for existing discussions before posting your own. https://github.com/ppy/osu/discussions?discussions_q=is%3Aopen+combo+burst
